epicure-frontend/src/shared/components/SearchBar/SearchBar.scss

```diff
@@ -4,7 +4,7 @@
 .search-bar {
   display: flex;
   align-items: center;
-  border-radius: 1.0vw;
+  border-radius: 0.5vw;
```

Comment:
why is the border radius needs to be effected by the width of the screen

Reply:
done